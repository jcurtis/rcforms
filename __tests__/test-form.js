import React from 'react'
import renderer from 'react-test-renderer'
import TestForm from '../__storybook__/test-form'

test('renders TestForm', () => {
  const component = renderer.create(
    <TestForm />
  )
  let tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
