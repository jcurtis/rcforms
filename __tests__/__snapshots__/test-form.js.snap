// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders TestForm 1`] = `
<section>
  <form
    className="form"
    noValidate={true}
    role="form"
  >
    <h1>
      My Test Form
    </h1>
    <p>
      <label
        htmlFor="input1"
      >
        Input 1
      </label>
      <input
        name="input1"
        onBlur={[Function]}
        onChange={[Function]}
        type="text"
        value=""
      />
    </p>
    <p>
      <label
        htmlFor="input2"
      >
        Input 2
      </label>
      <input
        name="input2"
        onBlur={[Function]}
        onChange={[Function]}
        type="text"
        value="initial2"
      />
    </p>
    <p>
      <label
        htmlFor="input3"
      >
        Input 3
      </label>
      <input
        name="input3"
        onBlur={[Function]}
        onChange={[Function]}
        type="text"
        value="initial3"
      />
    </p>
    <p>
      <label
        htmlFor="input4"
      >
        Input 4
      </label>
      <input
        name="input4"
        onBlur={[Function]}
        onChange={[Function]}
        type="text"
        value="initial4@test.com"
      />
    </p>
    <p>
      <label
        htmlFor="input5"
      >
        Input 5
      </label>
      <input
        name="input5"
        onBlur={[Function]}
        onChange={[Function]}
        type="text"
        value="initial5@test.com"
      />
    </p>
    <button
      onClick={[Function]}
    >
      Submit
    </button>
  </form>
</section>
`;
